### PR TITLE
document PuppetDB integration limitations

### PR DESCRIPTION
**Pull request by mmoll**
See theforeman/puppet-puppet#361 and theforeman/foreman-installer#196

### CODE REVIEW

File: _includes/manuals/1.13/3.2.3_installation_scenarios.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ foreman-installer \
   --puppet-server-storeconfigs-backend=puppetdb
 {% endhighlight %}
 
-Be aware that foreman-installer does not setup the PuppetDB server itself.
+Be aware that foreman-installer does not setup the PuppetDB server itself. Starting with 1.13 of foreman-installer, only
+setups using PuppetLabs Puppet 4 AIO packages are supported for PuppetDB integration using these parameters..
```
Comment (domcleal) [inline _includes/manuals/1.13/3.2.3_installation_scenarios.md]: Extra full stop at the end of the sentence should be removed, and Puppet Labs should be two words (or even "Puppet" now it's renamed).

File: _includes/manuals/1.13/1.2_release_notes.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ Installer modules have been updated, see their respective changelogs for more de
 *A full list of changes in 1.13.0 is available via [Redmine](http://projects.theforeman.org/rb/release/160)*
 
 ### Upgrade warnings
+* The installer support for [configuring Puppet masters to use a PuppetDB](/manuals/{{page.version}}/index.html#puppetdb-integration)
+  server is now limited to setups using PuppetLabs' Puppet 4 AIO packages.
```
Comment (domcleal) [inline _includes/manuals/1.13/1.2_release_notes.md]: ditto, "Puppet Labs" or "Puppet"

**Comment by mmoll** [comment]
updated

**Comment by domcleal** [comment]
Thanks @mmoll!

**Pull request merged**